### PR TITLE
Feature/multi language

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/literals.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    search: 'Search',
+    selectAll: 'Select all',
+    unselectAll: 'Unselect all'
+  },
+  es: {
+    search: 'Buscar',
+    selectAll: 'Seleccionar todo',
+    unselectAll: 'Deseleccionar todo'
+  },
+  fr: {
```
Comment (alberss) [inline src/literals.ts]: What about using ISO code? en-US, es-ES, fr-FR 😁

File: src/components/Provider/index.tsx [hunk 2]
```diff
@@ -3,15 +3,19 @@ import { ThemeProvider } from 'styled-components'
...
 import { ITheme } from '../../models'
+import LanguageContext from './LanguageContext'
 
 interface IProps {
   theme?: ITheme
+  language?: string
 }
 
-const Provider: React.FC<IProps> = ({ theme = defaultTheme, children }) => (
+const Provider: React.FC<IProps> = ({ theme = defaultTheme, language = 'en', children }) => (
```
Comment (alberss) [inline src/components/Provider/index.tsx]: Change en to ISO code en-US

**Review by piny4man — CHANGES_REQUESTED** [review]

File: src/literals.ts (hunk 1 above)
Comment (mfmontenegro) [reply src/literals.ts]: Changed!

File: src/components/Provider/index.tsx (hunk 2 above)
Comment (mfmontenegro) [reply src/components/Provider/index.tsx]: Done

**Review by piny4man — APPROVED** [review]

**Pull request merged**